templates/pages/account/edit-payment-method.html

```diff
@@ -7,49 +7,65 @@ <h2 class="page-heading">{{lang 'forms.payment_methods.edit.heading'}}</h2>
-                <div class="form-field">
-                    <input type="checkbox" value="default_instrument" name="default_instrument" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if customer.payment_methods.selected_payment_method.default_instrument}}checked{{/if}}>
-                    <label class="form-label" for="default_instrument">{{lang 'forms.payment_methods.default_instrument'}}</label>
+        {{#with customer.edit_stored_instrument}}
+            <form action="{{forms.action}}" data-address-form class="form" method="post">
+                <input type="hidden" name="token" value="{{token}}">
+                <input type="hidden" name="currency_code" value="{{../currency_selector.active_currency_code}}">
+                <input type="hidden" name="type" value="{{type}}">
+
+                <h3 class="paymentMethodForm-heading">{{lang 'account.payment_methods.payment_method'}}</h3>
+
+                <div class="paymentMethodForm">
+                    {{#if type '===' 'stored_card'}}
+                        <dl class="paymentMethodForm-details">
+                            <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.credit_card_number'}}</dt>
+                            <dd class="paymentMethodForm-details-description">**** **** **** {{last_4}}</dd>
+                        </dl>
+                        <dl class="paymentMethodForm-details">
+                            <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.expiration'}}</dt>
+                            <dd class="paymentMethodForm-details-description">{{expiry_month}}/{{expiry_year}}</dd>
+                        </dl>
+                    {{/if}}
+                    {{#if type '===' 'stored_paypal_account'}}
+                        <div class="form-field">
+                            <h3 class="paymentMethodForm-heading">
+                                {{lang 'account.payment_methods.paypal'}} {{lang 'common.account'}}
+                            </h3>
+                            <p>
+                                {{email}}
+                            </p>
+                        </div>
+                    {{/if}}
+                    <div class="form-field">
+                        <input type="checkbox" value="default_instrument" name="is_default" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if is_default}}checked{{/if}}>
+                        <label class="form-label" for="default_instrument">{{lang 'forms.payment_methods.default_instrument'}}</label>
+                        <input type="checkbox" value="default_instrument" name="default_instrument" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if customer.payment_methods.selected_payment_method.default_instrument}}checked{{/if}}>
```

Comment:
Is this right - `if customer.payment_methods.selected_payment_method.default_instrument`?
I think selected_payment_method is not part of the shape for V2.